src/mudlet.cpp

```diff
@@ -2411,6 +2441,8 @@ void mudlet::assignKeySequences()
 
         delete reconnectShortcut.data();
         dactionReconnect->setShortcut(reconnectKeySequence);
+
+        dactionCloseProfile->setShortcut(closeProfileKeySequence);
```

Comment:
You need extra stuff to accomplish the same effect as the others. (Part 2 of 2):
```suggestion
        delete closeShortcut.data();
        dactionCloseProfile->setShortcut(closeProfileKeySequence);
```

Reply:
Fixed